lib/service/Controller/Scout/LSPController.dart

```diff
@@ -57,12 +57,19 @@ class LSPController extends GetxController {
     }
   }
 
-  Future<void> redeem() async {
+  Future<bool> redeem() async {
     EthereumAddress address = EthereumAddress.fromHex(aptAddress.value);
     genericLSP = LongShortPair(address: address, client: tokenClient);
     final theCredentials = controller.credentials;
     BigInt tokensToRedeem = normalizeInput(redeemAmt.value);
     genericLSP.redeem(tokensToRedeem, credentials: theCredentials);
+
+    try {
+      await genericLSP.redeem(tokensToRedeem, credentials: theCredentials);
+      return true;
+    } catch (_) {
+      return false;
+    }
```

Comment:
calling the redeem function twice will have two redeem transactions in queue inside metamask. We only want to have one transaction in queue. Remove the one that is outside of the try catch block.

Reply:
Yea. will do